packages/apt/test.sh

```diff
@@ -67,6 +67,9 @@ MAKEFLAGS=-j$(nproc) gem install grntest
 if groonga --version | grep -q apache-arrow; then
   apt install -V -y \
     g++
```

Comment:
うーん、`libre2-dev`を事前にインストールしておかないとダメかも。`gem install red-arrow`で`libarrow-glib-dev`経由で自動インストールされるからこのタイミングではまだ存在しないかも。

```suggestion
    g++ \
    libre2-dev
```

Reply:
おぉ。なるほど。たしかに。